File: web3rpc/rpc-specs/code-samples/python/eth/block/getUncleByBlockNumberAndIndex.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

blockTag = "0xe8"
uncleIndex = "0x1"

w3 = Web3(Web3.HTTPProvider(host))
eth_response = w3.eth.get_uncle_by_block_number_and_index(blockTag, uncleIndex)

print(eth_response)

File: web3rpc/rpc-specs/code-samples/javascript/eth/block/getUncleByBlockNumberAndIndex.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");
  const blockNumber = 119189116;
  const uncleIndex = "0x1";

  provider.eth
    .getUncleByBlockNumberAndIndex(
      blockNumber,
      uncleIndex,
      {},
      (err, data, response) => {},
    )
    .then((data) => {
      console.log(data);
    });
})();
